components/mpas-seaice/cime_config/buildnml

```diff
@@ -214,12 +214,12 @@ def buildnml(case, caseroot, compname):
         decomp_date += '200908'
         decomp_prefix += 'mpas-seaice.graph.info.'
     elif ice_grid == 'WC14to60E2r3':
-        grid_date += '200714'
+        grid_date += '210210'
         grid_prefix += 'seaice.WC14to60E2r3'
         decomp_date += '200714'
         decomp_prefix += 'mpas-seaice.graph.info.'
     elif ice_grid == 'WC14to60E2r3_ICG':
-        grid_date += '201002'
+        grid_date += '210210'
```

Comment:
I couldn't find this one either?

Reply:
oops, forgot to move it.  Should be there now.

Reply:
Yes, all good. Thanks